pkg/encryption/path_new.go

```diff
@@ -0,0 +1,289 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package encryption
+
+import (
+	"crypto/hmac"
+	"crypto/sha512"
+	"encoding/base64"
+	"strings"
+
+	"github.com/zeebo/errs"
+	"storj.io/storj/pkg/paths"
+	"storj.io/storj/pkg/storj"
+)
+
+// StoreEncryptPath encrypts the path using the provided cipher and looking up
+// keys from the provided store and bucket.
+func StoreEncryptPath(bucket string, path paths.Unencrypted, cipher storj.Cipher, store *Store) (
```

Comment:
why aren't these methods on a store?

Reply:
Legacy. It can move there. Can we do that during when we do the big refactor to make it easier to move from this to that?

Reply:
:+1: